ci/osp18_functional_tests.yml

```diff
@@ -5,6 +5,7 @@
   environment:
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
+    JUNIT_TEST_CASE_PREFIX: "testsuites"
```

Comment:
Can we use "[test]" instead?

Reply:
@elfiesmelfie Yes we can .. 
but defining "JUNIT_TEST_CASE_PREFIX" here is not taking effect.. When I try to set it using "export JUNIT_TEST_CASE_PREFIX" locally on my test machine, it is working good